src/components/ebay-carousel/index.js

```diff
@@ -88,7 +88,7 @@ function getTemplateData(state) {
 
     items.forEach((item, i) => {
         const { style, transform } = item;
-        const marginRight = i !== items.length && `${gap}px`;
```

Comment:
This is technically unrelated, but the intent of this code was to remove the unneeded additional margin from the end which is was not doing because of the lack of the `-1`. This seemed like a extremely low risk change that didn't warrant an issue or another PR.